posthog/tasks/tasks.py

```diff
@@ -799,6 +799,13 @@ def update_survey_iteration() -> None:
     update_survey_iteration()
 
 
+@shared_task(ignrore_result=True)
```

Comment:
```suggestion
@shared_task(ignore_result=True)
```

not sure what this decorator does, but there's a typo in `ignore_result`, so my guess is that it won't do what you want ;)